packages/UI/.storybook/main.ts

```diff
@@ -33,9 +33,9 @@ const config: StorybookConfig = {
         esModuleInterop: false,
       },
       shouldExtractLiteralValuesFromEnum: true,
-      propFilter: () => true,
+      propFilter: (prop) => (prop.parent ? !/node_modules/.test(prop.parent.fileName) : true),
     },
-    reactDocgen: 'react-docgen',
+    reactDocgen: 'react-docgen-typescript',
```

Comment:
`react-docgen-typescript` is more advanced so I think we should keep `react-docgen-typescript` in